frontend/src/diagram/DiagramWebSocketContainer.tsx

```diff
@@ -617,6 +631,52 @@ export const DiagramWebSocketContainer = ({
     }
   };
 
+  const handleError = useCallback(
+    (loading, data, error) => {
+      let errorMessage = undefined;
+      if (!loading) {
+        if (error) {
+          errorMessage = error.message;
```

Comment:
You are not doing anything if there is an error right?

Reply:
I just show the error message for now.